src/App.tsx

```diff
@@ -1,12 +1,69 @@
-import { Box, Typography } from "decentraland-ui2"
+import { useState } from "react"
+import { ThemeProvider, dark } from "decentraland-ui2/dist/theme"
+import {
+  Box,
+  CircularProgress,
+  CssBaseline,
+  Toolbar,
+  Typography,
+} from "@mui/material"
+import Sidebar from "./components/Sidebar"
+import SquidsTable from "./components/SquidsTable"
+import TopBar from "./components/TopBar"
+import { useSquids } from "./hooks/useSquids"
+
+const drawerWidth = 240
 
 const App = () => {
+  const { squids, loading, error } = useSquids()
+  const [mobileOpen, setMobileOpen] = useState(false)
+
+  const handleDrawerToggle = () => {
+    setMobileOpen(!mobileOpen)
+  }
+
   return (
-    <Box>
-      <Typography variant="h4">
-        This is a Decentraland dApps template
-      </Typography>
-    </Box>
+    <ThemeProvider theme={dark}>
+      <Box sx={{ display: "flex", height: "100vh" }}>
+        <CssBaseline />
```

Comment:
```suggestion

```

The [ThemeProvider](https://github.com/decentraland/ui2/blob/master/src/theme/provider.tsx) already has the component: `<CssBaseline />`